packages/web/src/common/models/Analytics.ts

```diff
@@ -1175,6 +1181,28 @@ type SocialProofError = {
   error: string
 }
 
+type AudiusOauthStart = {
+  eventName: Name.AUDIUS_OAUTH_START
+  redirectUriParam: string | string[]
```

Comment:
Just wondering how it could be string[]?

Reply:
i think it'd be if you did something like `?foo=bar&foo=qux` or `?foo[]=bar&foo[]=qux`